R/tm_g_lineplot.R

```diff
@@ -294,6 +302,21 @@ tm_g_lineplot <- function(label,
                           pre_output = NULL,
                           post_output = NULL,
                           ggplot2_args = teal.widgets::ggplot2_args()) {
+  if (lifecycle::is_present(strata)) {
+    warning(
+      "The `strata` argument of `tm_g_lineplot()` is deprecated as of teal.modules.clinical 0.9.1. ",
+      "Please use the `group_var` argument instead.",
+      call. = FALSE
+    )
+    group_var <- strata
+  }
+
+  # Now handle 'parentname' calculation based on 'group_var'
+  parentname <- ifelse(
+    inherits(group_var, "data_extract_spec"),
+    teal.transform::datanames_input(group_var),
+    "ADSL"
+  )
```

Comment:
Bypass default if specified by user
```suggestion
  parentname <- if (is.null(parentname) {
    ifelse(
      inherits(group_var, "data_extract_spec"),
      teal.transform::datanames_input(group_var),
      "ADSL"
    )
  }
```